schema_version: 1.2.0
id: CVE-2017-5601
aliases: []
modified: 2018-11-30T11:29Z
published: 2017-01-27T22:59Z
details: An error in the lha_read_file_header_1() function (archive_read_support_format_lha.c)
  in libarchive 3.2.2 allows remote attackers to trigger an out-of-bounds read memory
  access and subsequently cause a crash via a specially crafted archive.
severity:
- type: CVSS_V3
  score: 7.5
- type: CVSS_V2
  score: 5.0
affected:
- package:
    name: libarchive
    cpe: cpe:2.3:a:libarchive:libarchive
- ranges:
  - type: GIT
    repo: https://github.com/libarchive/libarchive
    events:
    - introduced: '0'
    - fixed: 98dcbbf0bf4854bf987557e55e55fff7abbf3ea9
- versions:
  - 3.2.2
- database_specific:
    CWE:
    - CWE-125
    CVSS:
      Score: 7.5
      Severity: HIGH
      Code: CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H
references:
- type: WEB
  url: http://www.securityfocus.com/bid/95837
- type: WEB
  url: http://www.securitytracker.com/id/1037974
- type: WEB
  url: https://lists.debian.org/debian-lts-announce/2018/11/msg00037.html
- type: FIX
  url: https://github.com/libarchive/libarchive/commit/98dcbbf0bf4854bf987557e55e55fff7abbf3ea9
- type: WEB
  url: https://secunia.com/secunia_research/2017-3/
